src/index.tsx

```diff
@@ -1,3 +1,4 @@
+import 'preact/debug';
```

Comment:
Hi, pls remove it. `@preact-preset-vite` already handles it and removes it in prod